examples/md-flexible/input/explodingLiquid.yaml

```diff
@@ -34,8 +34,9 @@ Objects:
   CubeClosestPacked:
     0:
       particle-type-id           :  0
-      box-length                 :  [15, 6, 15]
-      bottomLeftCorner           :  [0, 27, 0]
+      box-length                 :  [14, 6, 14]
+      # Move particles half spacing away from domain border to prevent automatic border extension
+      bottomLeftCorner           :  [0.5, 27, 0.5]
```

Comment:
why is this required? and how do we communicate this to user whether or not this is required?

Reply:
> Why is this required?

When you specify any object in md-flex the domain is enlarged to fit it if required. For e.g. the grid generator, the actual box size is slightly larger than the grid box (namely half the spacing in each direction) so that it is perfectly aligned with its periodic images.

Usually all of this happens automatically no action required. However if you restart form a checkpoint you have to set the box size manually. I slightly moved the generator box so that the actual domain starts at `[0,0,0]` vs `[-.5, -.5, -.5]` as before.

> Who do we communicate this?

When a simulation starts it lists the parameters with which it started. This includes the new domain box. Again there is no specific action needed just careful input.